vars/piperStageWrapper.groovy

```diff
@@ -99,7 +99,11 @@ private void executeStage(script, originalStage, stageName, config, utils) {
     } finally {
         //Perform stashing of selected files in workspace
         utils.stashList(script, script.commonPipelineEnvironment.configuration.stageStashes?.get(stageName)?.stashes ?: [])
-        deleteDir()
+        //NOTE: We do not delete the directory because it failed on JaaS because of infrastructure issues.
```

Comment:
Isn't JAAS an internal term? Should we we use internal terms outside?

Reply:
Nope we shouldn't. I could have sworn I've read "JaaS" before and wasn't aware of the details. Changed the comment in a follow-up commit.